### ISSUE TITLE
Drag events do not fire in edit mode

### ISSUE THREAD

**Issue by catbadger** [Leaflet/Leaflet.draw#589]
I'm using the 1.0rc3 code in master, and I've set a polygons to always be in edit mode. draw:dragend and draw:drawstop are not firing at all though. Note also that I've hidden the edit button as my polygons are always in edit mode but even when i use the edit button, the polygon goes into a mode with dotted lines, but the events refuse to fire during edit.

**Update**: if I enable the edit button, the draw:edited event fires. this is a workaround, but will have to do for now.

**TLDR**: draw:drawstop is not firing at all when it is supposed to be

**Comment by catbadger** [comment]
we can bind the edit event to the polygon layer. this does what I'm after
